packages/hop-node/src/cli/confirmRoot.ts

```diff
@@ -0,0 +1,87 @@
+import ArbitrumBridgeWatcher from 'src/watchers/ArbitrumBridgeWatcher'
+import GnosisBridgeWatcher from 'src/watchers/GnosisBridgeWatcher'
+import OptimismBridgeWatcher from 'src/watchers/OptimismBridgeWatcher'
+import PolygonBridgeWatcher from 'src/watchers/PolygonBridgeWatcher'
+import { actionHandler, parseBool, parseInputFileList, parseString, parseStringArray, root } from './shared'
+import { getXDomainMessageRelayWatcher } from 'src/watchers/watchers'
+import { BigNumber } from 'ethers'
+
+type ExitWatcher = GnosisBridgeWatcher | PolygonBridgeWatcher | OptimismBridgeWatcher | ArbitrumBridgeWatcher
+type ConfirmRootData = {
```

Comment:
This type can be imported from `xDomainMessageRelayWatcher`

```suggestion
import { ConfirmRootData } from 'src/watchers/xDomainMessageRelayWatcher'
```